src/averageAges.js

```diff
@@ -14,12 +14,23 @@
  *
  * @return {number}
  */
+function averageAge(people) {
+  return people.reduce(
+    (acc, curr) => acc + (curr.died - curr.born), 0
+  ) / people.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(
+      person => person.sex === 'm'
+        && Math.ceil(person.died / 100) === century
+    )
+    : people.filter(
+      person => person.sex === 'm'
+    );
```

Comment:
Let's simplify this code, why to call filter in both cases, DRY, just choose needed condition inside one filter method